ChunkyDocs/docs/plugins/plugin_list.md

```diff
@@ -80,13 +88,13 @@ This plugin adds AI denoiser functionality using Intel Open Image Denoise. It is
 
 - Step 2: Download the <a href="https://www.openimagedenoise.org/downloads.html" target="_blank">Precompiled Intel Open Image Denoise Binary Packages</a> for your OS. (for example, on Windows, it would be "oidn-1.4.3.x64.vc14.windows.zip".)
 
-- Step 3: Extract the OIDN ZIP file to a location on your computer, such as "C:\Program Files\oidn-1.4.1.x64.vc14.windows".
+- Step 3: Extract the OIDN ZIP file to a location on your computer, such as "C:\Program Files\oidn-1.4.1.x64.vc14.windows". Alternatively, you may optionally only extract "oidnDenoise.exe", "OpenImageDenoise.dll", and "tbb12.dll" to a choisen directory. These are the minimum required files at the time of writing.
 
 - Step 4: Launch Chunky.
 
 - Step 5: Open the <samp>Denoiser</samp> tab in the left control panel.
 
-- Step 6: Click the <samp>...</samp> button, and then browse for "oidnDenoise.exe", which is located in the "bin" folder of the extracted OIDN ZIP file.
+- Step 6: Click the <samp>...</samp> button, and then browse for "oidnDenoise.exe", which is typically located in the "bin" folder of the extracted OIDN ZIP file.
```

Comment:
Are there any other places where the binary may be located?

Reply:
The user is able to extract the binary (and required .dll) to what ever folder they want. I suppose it might end up in "downloads" if they click the extract button however we might wish to encourage people to avoid this. Thoughts?